javascript/reactjs-todo/.env.example

```diff
@@ -1,7 +1,8 @@
 # APP_URL=$APP_URL # not using this for preview-environment instead, we can use window.location.origin
 AM_URL=$AM_URL
+SCOPE=$SCOPE
 API_URL=$API_URL
-DEBUGGER_OFF=true
+DEBUGGER_OFF=false
```

Comment:
We probably want this defaulted to OFF which means the value is `true`. How you have it now should result in all the `debugger;` statements pausing execution when the devtools is open. This can be a bit overwhelming if you don't expect it.

Reply:
fixed that. Somehow that was broken and `false` was actually `true` and vise versa